src/no-bake-oatmeal-cookies.md

```diff
@@ -0,0 +1,34 @@
+# No Bake Oat Bars
+
+This is a simple no-bake oat bars that you can make with
+just three simple ingredients. It's also quick and simple to make
+making it a great snack for when you have some ingredients lying
+around.
+
+- Prep Time: 20 mins
+- Cook Time: 10 mins
+- Servings: 4
+
+## Ingredients
+
+- 1/2 Cup Creamy Peanut Butter
+- 1/4 Cup Honey
+- 1 1/2 Cup Rolled Oats
```

Comment:
If you are going to quantify # of servings per ingredient serving size do it for all ingredients plz ty.

Reply:
Hello,

Sorry, I didn't seem to understand but what do you mean by "quantify # of servings per ingredient serving size do it for all ingredients"?

I believe I've added the quantity of ingredients needed to make at least four servings of the snack. Though I might just be misunderstanding the comment.

I hope to get some clarification on this.

Thanks!

Reply:
Hi DGK, are you commenting on the fact that it says "1½ cups"?  That's not saying you need one half-cup. It's mixed fraction notation, it means one and a half cup.

1½ = 1+½ = 3/2

Reply:
Ah, I see. @Demonstrandum